assets/src/domain/eventEditor/services/apollo/initialization/useInitQueries.ts

```diff
@@ -8,22 +8,22 @@ const useInitQueries = (): void => {
 	// initiate datetime fetching.
 	useFetchDatetimes();
 
-	// initiate ticket fetching.
+	// // initiate ticket fetching.
```

Comment:
???

Reply:
had commented smth out to test smth, it's cleaned up now